src/server/README.md

```diff
@@ -728,6 +728,64 @@ Student API
                 -   Unauthorized user -> Status code: 401
                 -   Internal server error -> Status code: 500
 
+    -   api/student/search
+    
+        -   Filter student records based on query and return them.
+        -   Body: None
+        -   Parameters:
+            ```
+            {
+                firstName?: string;
+                lastName?: string;
+                email?: string;
+                sid?: string;
+                departmentIds?: string[];
```

Comment:
I don't think departmentIds needs to be optional. Getting it as an empty array means the user didn't pick anything for it. so update to   departmentIds: string[];

Reply:
Done.